devtools/makefile_base_posix.mak

```diff
@@ -247,7 +247,7 @@ VSIGN ?= true
 
 ifeq ($(TARGET_PLATFORM),linux32)
 	# FIXME(maximsmol): linux can't link stripped .ar archives apparently
-	STRIP=""
+	STRIP="true"
```

Comment:
How did you fix missing symbols in `tier0.so`? (`__wrap_freopen` and friends in particular)

Also FYI Valve does not strip Linux binaries (run `readelf -s` on any of their binaries)

Reply:
This does not *enable* strip, but rather run `true` instead of `strip`, which will do nothing. This fixes the annoying `bash: : command not found` errors.